guide/02-api-overview/release_notes_240.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
<p>Under <strong>Map</strong> class there is typo: <u>decicated </u>should be <u>dedicated</u></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
:)